impeller/aiks/canvas.cc

```diff
@@ -226,6 +226,28 @@ bool Canvas::AttemptDrawBlurredRRect(const Rect& rect,
   return true;
 }
 
+void Canvas::DrawLine(const Point& p0, const Point& p1, const Paint& paint) {
+  if (paint.stroke_cap == Cap::kRound) {
```

Comment:
Nit: file an issue for round caps

Reply:
Done: https://github.com/flutter/flutter/issues/138254